presto-docs/src/main/sphinx/presto_cpp/properties.rst

```diff
@@ -113,41 +142,126 @@ The configuration properties of Presto C++ workers are described here, in alphab
+* **Type:** ``string``
+* **Default value:** ``4GB``
 
   Specifies the total amount of memory in GB reserved for the queries on
   a worker node. A query can only allocate from this reserved space if
   1) the non-reserved space in ``query-memory-gb`` is used up; and 2) the amount
-  it tries to get is less than ``memory-pool-reserved-capacity``.
+  it tries to get is less than ``shared-arbitrator.memory-pool-reserved-capacity``.
 
-``runtime-metrics-collection-enabled``
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-* **Type:** ``boolean``
+``shared-arbitrator.memory-pool-initial-capacity``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``string``
+* **Default value:** ``128MB``
+
+  The initial memory pool capacity in bytes allocated on creation.
+
+``shared-arbitrator.global-arbitration-enabled``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``string``
 * **Default value:** ``false``
 
-  Enables collection of worker level metrics.
+  If true, it allows shared arbitrator to reclaim used memory across query
+  memory pools.
 
-``system-memory-gb``
-^^^^^^^^^^^^^^^^^^^^
+``shared-arbitrator.memory-pool-reserved-capacity``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-* **Type:** ``integer``
-* **Default value:** ``40``
+* **Type:** ``string``
+* **Default value:** ``64MB``
 
-  Memory allocation limit enforced via internal memory allocator. It consists of two parts:
-  1) Memory used by the queries as specified in ``query-memory-gb``; 2) Memory used by the
-  system, such as disk spilling and cache prefetch.
+  The amount of memory in bytes reserved for each query memory pool. When
+  a query tries to allocate memory from the reserved space whose size is
+  specified by 'shared-arbitrator.reserved-capacity', it cannot allocate
+  more than the value specified in 'shared-arbitrator.memory-pool-reserved-capacity'.
 
-  Set ``system-memory-gb`` to the available machine memory of the deployment.
+``shared-arbitrator.memory-pool-transfer-capacity``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-``task.max-drivers-per-task``
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+* **Type:** ``string``
+* **Default value:** ``32MB``
 
-* **Type:** ``integer``
-* **Default value:** ``number of hardware CPUs``
+  The minimal memory capacity in bytes transferred between memory pools
+  during memory arbitration.
 
-  Number of drivers to use per task. Defaults to hardware CPUs.
+``shared-arbitrator.memory-reclaim-max-wait-time``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``string``
+* **Default value:** ``5m``
+
+  Specifies the max time to wait for memory reclaim by arbitration. The
+  memory reclaim might fail if the max wait time has exceeded. If it is
+  zero, then there is no timeout.
+
+``shared-arbitrator.fast-exponential-growth-capacity-limit``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``string``
+* **Default value:** ``512MB``
+
+  When shared arbitrator grows memory pool's capacity, the growth bytes will
+  be adjusted in the following way:
+   - If 2 * current capacity is less than or equal to
```

Comment:
`/Users/steveburnett/Documents/GitHub/presto/presto-docs/src/main/sphinx/presto_cpp/properties.rst:211: ERROR: Unexpected indentation.`


Adjust the indentation for this list until the gray vertical line in the screenshot does not appear in a local doc build. See [Building the documentation](https://github.com/prestodb/presto/tree/master/presto-docs#building-the-documentation) in the presto-docs README for information on local doc builds. 

<img width="806" alt="Screenshot 2024-08-22 at 2 47 20 PM" src="https://github.com/user-attachments/assets/bd621b9d-0a21-4ec2-8536-8ff78ef9e36d">

Reply:
Thanks, just revised